src/graph/visitor/DeducePropsVisitor.cpp

```diff
@@ -173,17 +177,66 @@ void DeducePropsVisitor::visit(VersionedVariableExpression *expr) { reportError(
 
 void DeducePropsVisitor::visit(LabelExpression *expr) { reportError(expr); }
 
-void DeducePropsVisitor::visit(AttributeExpression *expr) { UNUSED(expr); }
```

Comment:
Why delete this?

Reply:
properties($$).age is AttributeExpression, we should extract attribute of Vertex